audits/manifest/background-color.js

```diff
@@ -26,13 +26,16 @@ class ManifestBackgroundColor {
     return 'Contains background_color';
   }
 
-  static audit(inputs) {
-    let hasBackgroundColor = false;
-    const manifest = inputs.manifest;
+  static hasBackgroundColorValue(manifest) {
+    /* eslint-disable camelcase */
```

Comment:
Also in general I don't want us disabling our linter ad-hoc. If there's a problem with the rule we should resolve it, not disable it: either the rule isn't a good fit, or it is and we need to figure how to meet its requirements.